src/cmd/cmd.cpp

```diff
@@ -478,7 +478,12 @@ int main(int argc, char **argv) {
 
     SyncEngine engine(account, options.source_dir, folder, &db);
     engine.setIgnoreHiddenFiles(options.ignoreHiddenFiles);
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
+    QObject::connect(&engine, &SyncEngine::finished,
+                     [&app](bool result) { app.exit(result ? EXIT_SUCCESS : EXIT_FAILURE); });
```

Comment:
Jenkins does not seem to complain yet.
(but i dont know if it checks all platforms..)

Maybe where we have Qt5 we also have that?